### PR TITLE
Switch emojis to log-symbols for terminal compatibility

### LINKED ISSUE #362: Improved CLI Icons for Windows
🙋 Feature Request

### 🤔 Expected Behavior

Window's should fall back to icons that it supports and can render.

### 😯 Current Behavior

Window's renders colored boxes instead of ⌛️ ✨:

image

### 💁 Possible Solution

We could leverage something like [log-symbols](https://www.npmjs.com/package/log-symbols) to provide fallbacks. Alternatively, just detect Windows and conditionally show the existing ⌛️ ✨ vs other compatible icons.

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           |  N/A
| Node             |  N/A
| npm/Yarn         |  N/A
| Operating System |  Windows 10

### PR DESCRIPTION
**Pull request by shawwn**
Closes #362.

### CODE REVIEW

**Comment by shawwn** [comment]
@brandon93s Ok, I switched out the emojis with log-symbols. It'll be up to @devongovett whether he's fond of the existing hourglass / star emojis though.

Alternatively, we could use log-symbols on windows, and the current emojis everywhere else.

**Comment by brandon93s** [comment]
Yep. 

> Alternatively, just detect Windows and conditionally show the existing ⌛️ ✨ vs other compatible icons.

**Comment by devongovett** [comment]
yeah let's keep the emojis by default and fall back

**Comment by shawwn** [comment]
Ok, I'll preserve our emoji integrity.

#savemojis

**Comment by shawwn** [comment]
Yep, agreed. I just got distracted by holidays and focusing on impactful
work (like abstracting the FS layer). Feel free to submit your own PR if
you get time to tackle this, otherwise I'll eventually get around to it.

**Comment by devongovett** [comment]
Updated to use emoji but have fallback for windows. Apparently VS Code does support emoji though thanks to electron, so we try to detect that.

@brandon93s since you use windows, could you verify? Thanks!

**Comment by brandon93s** [comment]
@devongovett  - great find on the emoji support in VS Code! Windows test outputs below:

PowerShell:

image

CMD:

image

VSCode (PowerShell):

image

I've tested failures as well. All three states (processing, success, failure) look good.

File: src/utils/emoji.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const supportsEmoji = process.platform !== 'win32' || process.env.VSCODE_PID;
```
Comment (brandon93s) [inline src/utils/emoji.js]: 👍 I like the dedicated utility; should make maintenance easier!

**Comment by devongovett** [comment]
Awesome, thanks for testing @brandon93s! Merry Christmas. 🎄

**Comment by brandon93s** [comment]
Same to you! 🎄🎅🏼

**Pull request merged**